File: Evaluación (Alday-Olivares)/Empleado.java
public class Empleado extends Persona{
   
    // ATRIBUTOS
    private String codigo;
    private int sueldo_base;
    
    // SETTERS 
    public void setCodigo(String C)     {this.codigo = C;}
    public void setSueldo(int S)        {this.sueldo_base = S;}
    
    // GETTERS 
    public String getCodigo()   {return this.codigo;}
    public int getSueldo()      {return this.sueldo_base;}
    
    // CONSTRUCTORES
    public Empleado(){}
    public Empleado(String R, String N, String P, String M, String D, String T, String C, int S){
        super (R, N, P, M, D, T);
        this.codigo = C;
        this.sueldo_base = S;}
    
    // ESTADO - TOSTRING
    public String toString(){return super.toString()+
                                    "\nCódigo Empleado: "+this.codigo+
                                    "\nSueldo Base: "+this.sueldo_base;}
    
   
   
                                    
                                    
                                    
                                    
                                    
                                    
                                    
}

File: Evaluacion1SebastianAlday/Empleado.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Evaluacion1SebastianAlday {
    public class Empleado : Persona {

        // ATRIBUTOS 

        private string codigo = "";
        private double sueldo_base = 0;


        // CONSTRUCTOR
        protected Empleado() { }
        protected Empleado(string R, string N, string AP, string AM, string D, string T, string C, double SB) : base(R, N, AP, AM, D, T) {
            codigo = C;
            sueldo_base = SB;
        }


        // GETTERS 
        public string getCodigo() { return codigo; }
        public double getSueldoBase() { return sueldo_base; }


        // SETTERS 
        public void setCodigo(string C) { codigo = C; }
        public void setSueldoBase(double SB) { sueldo_base = SB; }
    }
}
